app/workers/unverified_commits_reminder_worker.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class UnverifiedCommitsReminderWorker
+  UNVERIFIED_FOR_DURATION = 3.hours
+
+  include Sidekiq::Worker
+  sidekiq_options retry: 3
+
+  def perform
+    user_emails = Commit.deployed_but_not_verified_for_duration(UNVERIFIED_FOR_DURATION).pluck(:email)
+    slack_usernames = User.where(email: user_emails)
+                          .where.not(slack_username: [nil, ""])
+                          .pluck(:slack_username)
+    return if slack_usernames.empty?
+
+    Rails.logger.info("Reminding #{slack_usernames.join(", ")} to verify their commits.")
+    message = "<@%s> Please verify your commits on Wilfred" % slack_usernames.join("> <@")
+    slack_client = Slack::Web::Client.new(token: ENV["SLACK_APP_ACCESS_TOKEN"])
+    slack_client.chat_postMessage(channel: ENV["SLACK_ROOM"], text: message, username: "Wilfred", as_user: false)
```

Comment:
Can replace this as follows similar to the usage found in `Commit#notify_user_to_verify`. Let me know which one you prefer.

```suggestion
    User.first.slack.chat_postMessage(channel: ENV["SLACK_ROOM"], text: message, username: "Wilfred", as_user: false)
```